GeoMesa_Feature_List.md

```diff
@@ -0,0 +1,100 @@
+> The following document uses a hierarchical structure of features - from high level descriptions of general functionality to the subfeatures of which those general functions are composed.
+
+
+## Data Ingest/Input
+
```

Comment:
we also support storm/kafka ingestion: http://www.geomesa.org/documentation/tutorials/geomesa-quickstart-storm.html